test/cmdlineargs.jl

```diff
@@ -97,6 +97,11 @@ let exename = `$(Base.julia_cmd()) --startup-file=no`
         @test startswith(read(`$exename --help`, String), header)
     end
 
+    # --project
+    let expanded = abspath(expanduser("~/foo"))
+        @test occursin(expanded, readchomp(`$exename --project='~/foo' -E 'Base.active_project()'`))
```

Comment:
Can you add another test here that sets `JULIA_PROJECT` instead of using `--project` and check that it works the same? Technically this file is for testing command line arguments, but it seems fine to test that here as well.

Reply:
I know in Linux one can do:
`env JULIA_PROJECT=~/foo ./julia  -E 'Base.active_project()' `

What is the OS-independent way of doing this?

Reply:
Yup: https://docs.julialang.org/en/v1/base/base/#Base.setenv

Reply:
Done. Tested works (i.e not throwing more errors than `p.exitcode`